bundles/binding/org.openhab.binding.zwave/database/evolve/lcd1.xml

```diff
@@ -0,0 +1,647 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Product>
+	<Model>EVLCD-1</Model>
+	<Label lang="en">LCD Keypad</Label>
+	<CommandClasses>
+		<Class>
+			<id>0x20</id>
+		</Class>
+		<Class>
+			<id>0x26</id>
+		</Class>
+		<Class>
+			<id>0x27</id>
+		</Class>
+		<Class>
+			<id>0x2b</id>
+		</Class>
+		<Class>
+			<id>0x2d</id>
+		</Class>
+		<Class>
+			<id>0x2c</id>
+		</Class>
+		<Class>
+			<id>0x43</id>
+		</Class>
+		<Class>
+			<id>0x70</id>
+		</Class>
+		<Class>
+			<id>0x72</id>
+		</Class>
+		<Class>
+			<id>0x73</id>
+		</Class>
+		<Class>
+			<id>0x85</id>
+		</Class>
+		<Class>
+			<id>0x86</id>
+		</Class>
+		<Class>
+			<id>0x87</id>
+		</Class>
+		<Class>
+			<id>0x92</id>
+		</Class>
+		<Class>
+			<id>0x93</id>
+		</Class>
+	</CommandClasses>
+	<Configuration>
+		<Parameter>
+			<Index>1</Index>
+			<Type>list</Type>
+			<Default>0</Default>
+			<Size>1</Size>
+			<Label lang="en">Button 1 Type</Label>
+			<Help lang="en">Sends Scene command to association group (Group 1 for
+				ON, Group 6 for OFF) when
+				un-inverted, sends BASIC_SET OFF to
+				association group when inverted.
+				Only one button can be defined as
+				privacy or housekeeping.
+			</Help>
+			<Item>
+				<Value>0</Value>
+				<Label lang="en">Scene Control Momentary</Label>
+			</Item>
+			<Item>
+				<Value>1</Value>
+				<Label lang="en">Scene Control Toggle</Label>
+			</Item>
+			<Item>
+				<Value>2</Value>
+				<Label lang="en">BASICSET Toggle</Label>
+			</Item>
+			<Item>
```

Comment:
Just checking that there shouldn't be an option 3? In param 2 there is and it looks basically the same as param 1...